cmd/utils/flags.go

```diff
@@ -805,7 +797,7 @@ func setBootstrapNodes(ctx *cli.Context, cfg *p2p.Config) {
 		urls = params.RinkebyBootnodes
 	case ctx.GlobalBool(GoerliFlag.Name):
 		urls = params.GoerliBootnodes
-	case cfg.BootstrapNodes != nil:
+	case cfg.BootstrapNodes != nil || cfg.BootstrapNodesV5 != nil:
```

Comment:
This seems dangerous. If either of the fields have been already set, you are skipping setting the other too. The original code had 2 methods, so you could skip setting v4 or v5 individually.